data/hai/hai reveal 4 middle.txt

```diff
@@ -76,15 +77,24 @@ mission "Hai Reveal [A02] Symbol Identification"
 			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff."`
```

Comment:
If you're about to launch into asking her about the thing you're meant to be finding out about *discretely*, there should at least be some stated separation from others.

```suggestion
			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff." As she speaks she directs you to some seats that are just far enough away from the other people in the room to confer at least the illusion of privacy.`
```